src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -30,6 +30,9 @@
 import com.google.common.collect.Iterables;
```

Comment:
We don't have any tests currently for input from standard in. That's why I didn't. I can, but it'll take some time and refactoring of AbstractCommandLineRunner to handle a provided input stream rather than the hardcoded "stdin" stream.